cli/command/configure/cmd.go

```diff
@@ -70,20 +52,46 @@ func interactiveConfiguration() (*credentials.Credentials, error) {
 	explainHowToObtainCredentials()
 	creds := getDefaultCredentials()
 
-	usernameQuestion := &survey.Input{
-		Message: "SauceLabs username",
-		Default: creds.Username,
-	}
-	accessKeyQuestion := &survey.Input{
-		Message: "SauceLabs access key",
-		Default: creds.AccessKey,
-	}
-	err := askNotEmpty(usernameQuestion, &creds.Username)
-	if err != nil {
-		return nil, err
+	qs := []*survey.Question{
+		{
+			Name: "username",
+			Prompt: &survey.Input{
+				Message: "SauceLabs username",
+				Default: creds.Username,
+			},
+			Validate: func(val interface{}) error {
+				str, ok := val.(string)
+				if !ok {
+					return errors.New("invalid input")
+				}
+				re := regexp.MustCompile(`^([a-zA-Z0-9-.]{3,})$`)
```

Comment:
Hm, where is that regex for usernames from?

Reply:
Hm, was a placeholder I forgot to annotate. I replace by the one available in `sauce-lib`.